### DISCUSSION TITLE
地下城 III更新（最好中文版）

### DISCUSSION THREAD

**Discussion by luqiyihui** [Gamuxorg/bbs#229] [投递和请求游戏]
游戏名称：地下城 III
游戏名称(英文)：Dungeons III
下载地址 :（如果有的话）：
版主：这个游戏能更新下吗？再能多整合点DLC，当然最好是中文版的。

要求好像有点多，抱歉！！

**Comment by wangling12** [comment]
暂时找不到带中文的版本，带俄语的倒是一大堆
